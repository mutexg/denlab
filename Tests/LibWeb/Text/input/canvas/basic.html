<script src="../include.js"></script>
<script>
    test(() => {
        let canvas = document.createElement("canvas");
        let context = canvas.getContext("2d");
        context.font = '20px DenLabSans';
        println(context.font);
        context.font = '!!!'; // Invalid value, should be ignored.
        println(context.font);
    });
</script>
