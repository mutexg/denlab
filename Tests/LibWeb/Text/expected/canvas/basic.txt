normal normal 20px DenLabSans
normal normal 20px DenLabSans
